examples/relations_with_unique.py

```diff
@@ -0,0 +1,57 @@
+"""
+This example shows how relations between models work.
+
+Key points in this example are use of ForeignKeyField and ManyToManyField
+to declare relations and use of .prefetch_related() and .fetch_related()
+to get this related objects
+"""
+import logging
+
+from tortoise import Tortoise, fields, run_async
+from tortoise.models import Model
+
+logging.basicConfig(level=logging.DEBUG)
+
+
+class School(Model):
+    uuid = fields.UUIDField(pk=True)
+    name = fields.TextField()
+    id = fields.IntField(unique=True)
```

Comment:
I like how you confused the issue here on purpose. Makes this a more robust test :-)

Reply:
Sure. I will add enough testsuite :)